test/fit_tests/config/global_config.json

```diff
@@ -89,5 +89,6 @@
     "_comment": "These are the northbound rest API port assignments",
     "http": 8080,
     "https": 443
-  }
+  },
+  "ovaoui": "00:50:56"
```

Comment:
Could you please add a comment in to explain what this variable is- "_comment": "xxx"
And is the value good for the world to use or just our lab?